Add readmes to notebooks (#184)
openvinotoolkit/openvino_notebooks: notebooks/202-vision-superresolution/README.md

# Super Resolution with OpenVINO tutorial

image | video |
|---|---|
[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F202-vision-superresolution%2F202-vision-superresolution-image.ipynb) | [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/openvinotoolkit/openvino_notebooks/HEAD?filepath=notebooks%2F202-vision-superresolution%2F202-vision-superresolution-video.ipynb)

![tower](data/tower.jpg)

Super Resolution is the process of enhancing the quality of an image by increasing the pixel count using deep learning.

> ## Content table

Reviewer: ```suggestion
## Notebook Contents
```

> In this tutorial there will be two notebooks. Both of them are following similar ideas of using model called [Single Image Super Resolution (SISR)](https://github.com/openvinotoolkit/open_model_zoo/tree/develop/models/intel/single-image-super-resolution-1032) which is available from the Open Model Zoo. It is based on this [research paper]((https://arxiv.org/abs/1807.06779)).

Reviewer: ```suggestion
This tutorial is comprised of two notebooks. Both follow a similar concept of using a model called [Single Image Super Resolution (SISR)](https://github.com/openvinotoolkit/open_model_zoo/tree/develop/models/intel/single-image-super-resolution-1032) which is available from the Open Model Zoo. It is based on this [research paper]((https://arxiv.org/abs/1807.06779)).
```
